server/src/main/java/io/spine/server/enrich/EnricherBuilder.java

```diff
@@ -204,7 +203,7 @@ public boolean equals(Object obj) {
             if (!(obj instanceof Key)) {
                 return false;
             }
-            final Key other = (Key) obj;
+            final var other = (Key) obj;
             return Objects.equals(this.sourceClass, other.sourceClass)
                     && Objects.equals(this.enrichmentClass, other.enrichmentClass);
```

Comment:
I know this is out of scope for this PR, but can we refer to the class FQNs? `Class.equals(..)` works via `==` and thus is class loader-dependent.